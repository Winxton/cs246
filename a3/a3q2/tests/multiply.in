rahello
mb3awblbpcb2
wc
q
