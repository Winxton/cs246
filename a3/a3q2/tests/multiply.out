hellohellohello
15
hellohellohellohellohellohello
